Test/CoreSDK.Test/Shared/Extensibility/Implementation/TaskTimerTest.cs

```diff
@@ -153,7 +153,7 @@ public void CancelsPreviousActionWhenStartIsCalledMultipleTimes()
             }
 
             [TestMethod]
-            [Timeout(1000)]
+            [Timeout(1500)]
```

Comment:
it depends on something being executed on a threadpool thread  - and there is no guarantee esp. if TP started with minimum number... CodeCov tool is probably causing the additional time